test/e2e/storage/volume_provisioning.go

```diff
@@ -941,19 +938,29 @@ func getDefaultPluginName() string {
 	return ""
 }
 
-func newStorageClass(t testsuites.StorageClassTest, ns string, suffix string) *storagev1.StorageClass {
+func newStorageClass(t testsuites.StorageClassTest, ns string, prefix string) *storagev1.StorageClass {
 	pluginName := t.Provisioner
 	if pluginName == "" {
 		pluginName = getDefaultPluginName()
 	}
-	if suffix == "" {
-		suffix = "sc"
+	if prefix == "" {
+		prefix = "sc"
 	}
 	bindingMode := storagev1.VolumeBindingImmediate
 	if t.DelayBinding {
 		bindingMode = storagev1.VolumeBindingWaitForFirstConsumer
 	}
-	sc := getStorageClass(pluginName, t.Parameters, &bindingMode, ns, suffix)
+
+	// windows storage class can only be ntfs
+	if framework.NodeOSDistroIs("windows") {
```

Comment:
if this logic is useful for other tests too, maybe put it in a util function

Reply:
this logic only applied to windows? Seems like Linux should also use the same logic here?

Reply:
Correct, I removed the if statement but there was an issue with this test among others:

```
[sig-storage] Dynamic Provisioning GlusterDynamicProvisioner should create and delete persistent volumes [fast]

Warning  ProvisioningFailed  0s (x5 over 47s)  persistentvolume-controller  Failed to provision volume with StorageClass "volume-provisioning-7307-glusterdptestpjc72": invalid option "fstype" for volume plugin kubernetes.io/glusterfs
```

I was checking the parameters allowed in https://kubernetes.io/docs/concepts/storage/storage-classes/ and some of these StorageClasses don't accept `fsType` (one of them is glusterfs among others), before, the storage class fsType wasn't overriden from the default value so I think that we should preserve that, what if we add only the exception for windows?